### ISSUE TITLE
Could not start miner: API call not recognized: /miner/start

### ISSUE THREAD

**Issue by Zesseth** [NebulousLabs/Sia-GPU-Miner#59]
I have read that SIA do not automatically start miner anymore.

I try to run command 'miner start' on sia commandline.
I get this error message:  Could not start miner: API call not recognized: /miner/start

When i try to run sia-gpu-miner.exe I get error message HTTP error 404
I can not find any resources how this should be installed etc.

I have also try to start sia with command Sia-UI.exe -M "cgmtw", but it do not change behavior.
Firewall has exclusions for SIA and miner. 

Yep... I am running SIA with Windows 10.

**Comment by pdroalves** [comment]
Same problem here, but on Ubuntu 16.04 running siad 1.2.2.

**Comment by droghio** [comment]
There are a few things that can cause this issue, but these steps should get you up and running.

1. Quit Sia-UI (using the `Quit Sia` option shown by right-clicking the notification tray icon) and try starting the siad agent directly.
For Windows: `C:\Program Files (x86)\Sia-UI\resources\app\Sia\siad.exe --modules cgmhrtw -d %appdata%\Sia-UI\sia`

**NOTE:** It may take a few minutes for Siad to finish loading, it took six for me with an SSD.

2. Relaunch the Sia-UI (it will bind to your manually started siad instance)
3. Unlock your wallet
4. Retry the `miner start` command in the Terminal Plugin.

Assuming you get a `CPU Miner is now running.` message the last step is to retry the sia-gpu-miner.exe. Make sure you are running the latest version (v1.1.0).

The commands for Linux are similar but you will have to replace the Windows paths with those for your system.

**Comment by Zesseth** [comment]
Hi,

Great I get it working with your instructions!
Now mining...

**Comment by droghio** [comment]
Glad I could help. You might want to disable CPU mining using the `miner stop` command. It's up to you but for most systems it is not cost effective. You will still be able to GPU mine with the CPU miner stopped. Also try starting the sia-gpu-miner.exe with different -I values. No telling what will work best for your card, but -I 27 tends to work pretty well for most people.

Since your problem has been resolved I'm closing this issue.

Happy mining!

**Comment by Zesseth** [comment]
Thanks once more! I will try that setting.

Yeah, "code issue" is resolved, but I found behavior of SIA bit... not user friendly. This could be 'user story' for future development.

Also it could be documented. I believe many users do not use git and do not want to create account for git as they do not code.
I could see if I found time to write wiki or something for this.

Off topic:
Trying to get my first 2kSC to get in SIA... market is quite dry and hard to use for many "normal users"... this might come too big obstacle for many "normal users" and they will not stay to use SIA.
